src/js/wrapper.ts

```diff
@@ -566,11 +566,6 @@ export const NATIVE: SentryNativeWrapper = {
     if (level == 'log' as SeverityLevel) {
       return 'debug' as SeverityLevel;
     }
-    else if (level == 'critical' as SeverityLevel) {
-      return 'fatal' as SeverityLevel;
-    }
```

Comment:
Why this is removed? there's no possibility of being `critical` on the JS layer anymore?

Reply:
Yes, `critical` was removed.

https://github.com/getsentry/sentry-javascript/blob/0f9a3dfb864e8b0c57daac7dbda333abee20d04d/packages/types/src/severity.ts#L22